Objects/listobject.c

```diff
@@ -1044,6 +1045,193 @@ sortslice_advance(sortslice *slice, Py_ssize_t n)
 #define IFLT(X, Y) if ((k = ISLT(X, Y)) < 0) goto fail;  \
            if (k)
 
+/* Here we define custom comparison functions to optimize for the cases one commonly 
+ * in practice: homogeneous lists, often of one of the basic types. */
```

Comment:
Isn't there a "encounters" or "finds" missing here, between "commonly" and "in practice"?